platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/tipset/TipsetEventCreatorImpl.java

```diff
@@ -69,7 +69,11 @@ public class TipsetEventCreatorImpl implements TipsetEventCreator {
     private final ChildlessEventTracker childlessOtherEventTracker;
     private final TransactionSupplier transactionSupplier;
     private final SoftwareVersion softwareVersion;
-    /** The address book for the current network. */
+    private long minimumGenerationNonAncient = GENERATION_UNDEFINED + 1;
```

Comment:
`GraphGenerations.FIRST_GENERATION`
I know, the constants should be in a single place, but we can fix it in the future

Reply:
fixed